src/posts/2025-09-16-kernel-basics-for-tetragon/index.md

```diff
@@ -0,0 +1,226 @@
+---
+path: '/blog/2025/09/16/kernel-basics-for-tetragon'
+date: '2025-09-16T12:00:00.000Z'
+title: 'Linux Kernel Fundamentals for Writing Effective Tetragon Tracing Policies '
+ogImage: cover.png
+isFeatured: false
+ogSummary: 'Learn the basics of the Linux Kernel needed to effectively write good Tetragon tracing policies'
+categories:
+  - Community
+tags:
+  - Tetragon
+---
+
+**_Author: Paul Arah, Isovalent@Cisco_**
+
+![](cover.png)
```

Comment:
I find it difficult starting to read a blog post with a diagram, not sure what should I focus on and if I should come back later after reading. Maybe add a bit a context `we will come back to it` or move it later in the article.